spec.bs

```diff
@@ -3899,11 +3900,14 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|configIDL|)</dfn> m
     1. [=list/For each=] |seller config| in |configIDL|["{{AdAuctionDataConfig/sellers}}"]:
       1. Let |seller| be the result of running [=parse an https origin=] on
         |seller config|["{{AdAuctionOneSeller/seller}}"].
-      1. If |configs| already [=list/contains=] a [=auction data config=] whose
+      1. If |configs| [=list/contains=] an [=auction data config=] whose
          [=auction data config/seller=] is equal to |seller|, then
          [=exception/throw=] a {{TypeError}}.
-      1. Let |coordinator| be the result of running [=parse an https origin=] on
-        |seller config|["{{AdAuctionOneSeller/coordinatorOrigin}}"].
+      1. If |seller config|["{{AdAuctionOneSeller/coordinatorOrigin}}"] [=map/exists=]:
+        1. Let |coordinator| be the result of running [=parse an https origin=] on
+          |seller config|["{{AdAuctionOneSeller/coordinatorOrigin}}"].
+      1. Otherwise:
+        1. Let |coordinator| be the [=user agent=]'s default coordinator.

```

Comment:
same, be clearer about what coordinator it is.

Reply:
Done.